aws/policy/networking.yaml

```diff
@@ -71,6 +74,7 @@ Statement:
       - ec2:DescribeRouteTables
       - ec2:DescribeSecurityGroups
       - ec2:DescribeSubnets
+      - ec2:DescribeTags
```

Comment:
```suggestion
```

The EC2 Tags permissions are over in the compute policy and should already be there.

Reply:
ec2:DescribeTags here duplicates the ec2:DescribeTags in compute.yaml.  Amazon limits policy size (by character count) and we tend to run into those limits so it's helpful to avoid duplication.